manage-data/migrate/migrate-from-a-self-managed-cluster-with-a-self-signed-certificate-using-remote-reindex.md

```diff
@@ -1,6 +1,10 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud/current/ec-remote-reindex.html
+applies:
+  serverless: unavailable
+  hosted: all
+  ece: unavailable
 ---
 
 # Migrate from a self-managed cluster with a self-signed certificate using remote reindex [ec-remote-reindex]
```

Comment:
The title is quite long in the table of contents. You could consider adding a `navigation_title` to the frontmatter with a shorter title.

Reply:
Good idea! I've shortened it to:

`navigation_title: Reindex from a self-managed cluster`